src/OrbitGl/ThreadStateBar.cpp

```diff
@@ -161,17 +162,30 @@ std::string ThreadStateBar::GetThreadStateSliceTooltip(PrimitiveAssembler& primi
       static_cast<const ThreadStateSliceInfo*>(user_data->custom_data_);
   uint64_t begin_ns = thread_state_slice->begin_timestamp_ns();
   uint64_t end_ns = thread_state_slice->end_timestamp_ns();
-
-  return absl::StrFormat(
+  std::string thread_name = capture_data_->GetThreadName(thread_state_slice->woker_tid());
+  std::string process_name = capture_data_->GetThreadName(thread_state_slice->woker_pid());
+  std::string description = absl::StrFormat(
       "<b>%s</b><br/>"
       "<i>Thread state</i><br/>"
       "<br/>"
-      "%s"
+      "%s",
+      GetThreadStateName(thread_state_slice->thread_state()),
+      GetThreadStateDescription(thread_state_slice->thread_state()));
+  std::string woker_info;
+  if (thread_state_slice->woker_tid() != 0u) {
+    woker_info = absl::StrFormat(
```

Comment:
Nit: I would prefer `description += absl::StrFormat(...` instead of creating a separate variable. Same for `time` below.

(And then I would rename `description` to `tooltip` since this is now the whole tooltip.)